content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,121 @@
+
+### [Q: Is there a way to list all the experiments I have on my DVC remote that have not been committed to Git?](https://discord.com/channels/485586884165107732/563406153334128681/836705209039978538)
+
+Yes! The command `dvc exp list gitremote --all` lists all of the experiments for
+the default remote in the Git repo.
+
+Thanks again @Chandana for this gem!
+
+### [Q: Is CML compatible with Azure DevOps?](https://discord.com/channels/485586884165107732/728693131557732403/841664412221177926)
+
+Another great question from @Chandana!
+
+Right now, we support GitHub and GitLab.
+
+Azure DevOps and GCP (Google Cloud Platform) support are on the roadmap. Stay
+tuned for more updates!
+
+### [Q: I pushed a lot of files using `dvc push` to my DVC remote, but there are a few that couldn't be pushed at the time. If I run `dvc push` again, will it just upload the missing files?](https://discord.com/channels/485586884165107732/563406153334128681/842662337159757854)
+
+Thanks for the question @petek!
+
+Yes! You can just re-run `dvc push` and it will only upload the missing files.
+
+It might be a little slower than you would expect because DVC has to do some
+checks to make sure that the other files were uploaded successfully before, but
+as far as the actual data transfer goes, only the missing files will be
+uploaded.
+
+### [Q: Let's say I have a DVC pipeline with two stages, can I only pull the second one and keep the first one for other uses? Can I pull some specific output from the pipeline?](https://discord.com/channels/485586884165107732/485596304961962003/841688323663855616)
+
+You can pull specific outputs from a pipeline with
+`dvc pull path/to/specific/output`. This is similar to how you can use `dvc add`
+to work with specific files and directories.
+
+Thanks for such a great question @LucZ!
+
+### [Q: How does DVC handle incremental changes in the data and how does it work with non-DVC based pipeline features?](https://discord.com/channels/485586884165107732/485596304961962003/846364469524430848)
+
+These are good questions for common problems in MLOps from @Phoenix!
+
+To answer the first part, say you are getting new data every week. When you use
+DVC, you don't have to worry about getting duplicate data.
+
+DVC supports file-level deduplication right now, so if your data is in a shape
+of directory with files, then all unique files will only be stored once.
+Chunk-level deduplication is on our todo list.
+
+For the second part of the question, you can use data management with DVC and
+have your own pipelines. Just treat it as Git for data then be sure to
+`dvc {add,push,pull}` and you should be set. Hooks, like `pre-commit` or
+`post-pipeline-run`, are a good way to go about it.
+
+### [Q: Is there a way to tell DVC to use a different profile instead of the default profile when running S3 commands?](https://discord.com/channels/485586884165107732/563406153334128681/846857498094469120)
```

Comment:
` when running S3 commands` -> a bit misleading

when interacting with S3? or something like this ... can be better?